src/overlays/actors/ovl_En_Elf/z_en_elf.c

```diff
@@ -572,37 +572,35 @@ void func_80A03018(EnElf* this, GlobalContext* globalCtx) {
     Actor_MoveForward(&this->actor);
 }
 
-#ifdef NON_MATCHING
-// slight ordering and regalloc
 void func_80A03148(EnElf* this, Vec3f* arg1, f32 arg2, f32 arg3, f32 arg4) {
     f32 xVelTarget;
     f32 zVelTarget;
     f32 xzVelocity;
     f32 clampedXZ;
+    f32 temp;
 
```

Comment:
this temp was added but doesnt get used I believe. if it can be removed then do so, or make it an s32 pad if its needed for stack

Reply:
nah, I just forgot to take it out after testing.